ztp/source-crs/TunedPerformancePatch.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       data: |
```

Comment:
Can we explain a bit more in line12 to indicate the part that should match the PerformanceProfile name?

Reply:
Agreed.  A comment explaining that if you change the PerformanceProfile name you need to change this to match is probably important both here and in `PerformanceProfile.yaml`

Reply:
Added comment in both files.